packages/@aws-cdk/aws-pipes-alpha/lib/inputTransformation.ts

```diff
@@ -0,0 +1,55 @@
+
+enum reservedVariables {
```

Comment:
```suggestion
enum ReservedVariables {
```

Reply:
Renamed to `PipeVariable` and dropped the prefix.